ui/components/DashboardComponent.js

```diff
@@ -541,16 +542,8 @@ class DashboardComponent extends React.Component {
 
   handleError = (msg) => (error) => {
     this.props.updateProgress({ showProgress : false });
-    const self = this;
-    this.props.enqueueSnackbar(`${msg}: ${error}`, {
-      variant : "error", preventDuplicate : true,
-      action : (key) => (
-        <IconButton key="close" aria-label="Close" color="inherit" onClick={() => self.props.closeSnackbar(key)}>
-          <CloseIcon style={iconMedium} />
-        </IconButton>
-      ),
-      autoHideDuration : 7000,
-    });
+    const notify = this.props.notify;
```

Comment:
This is what I have done for the HOC. I have seen the file you have sent in the issue. Is there anything I need to change here?